libs/components/tabs/src/lib/modules/tabs/tabset.component.spec.ts

```diff
@@ -1574,4 +1593,52 @@ describe('Tabset component', () => {
       validateTabSelected(fixture.nativeElement, 0);
     }));
   });
+
+  describe('Permalinks with non-base URLs', () => {
+    let fixture: ComponentFixture<SkyTabsetPermalinksFixtureComponent>;
+    let location: Location;
+
+    beforeEach(() => {
+      location = TestBed.inject(Location);
+      fixture = TestBed.createComponent(SkyTabsetPermalinksFixtureComponent);
+    });
+
+    it('should not affect the base URL when opened or closed', fakeAsync(() => {
+      fixture.componentInstance.activeIndex = 0;
+      fixture.componentInstance.permalinkId = 'foobar';
+      fixture.componentInstance.router.navigate([
+        'example-path',
+        'example-child-path',
+      ]);
+
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+      tick();
```

Comment:
I know this was probably a copy/paste of another test, but I was able to get the test to pass with only one set of:
```
fixture.detectChanges();
tick();
```
(Same for the two blocks, below.)

Reply:
Good point. Thanks! Will remove the extras.